### ISSUE TITLE
City bombardment issues

### ISSUE THREAD

**Issue by Edaka** [LoneGazebo/Community-Patch-DLL#4566]
_1. Mod version (i.e Date - 4/23):_
6/14

_2. Mod list (if using Vox Populi only, leave blank):_

_3. Error description:_ 

Okay, I think I've searched before creating this one :)
So, this is how it looks in TechChanges.sql:
UPDATE Technologies SET BombardRange=1 WHERE Type='TECH_AGRICULTURE';
UPDATE Technologies SET BombardRange=2 WHERE Type='TECH_MATHEMATICS';
UPDATE Technologies SET BombardIndirect=1 WHERE Type='TECH_GUNPOWDER';
UPDATE Technologies SET BombardRange=3 WHERE Type='TECH_DYNAMITE';

By the looks of it, you're supposed to get:
1) range 2 without indirect fire at mathematics,
2) still range 2 (?) **with**  indirect fire at gunpowder 
3) range 3 **with** (?) indirect fire at dynamite.

However, in my experience, it works like this: 
1) range 2 with indirect fire at mathematics, 
2) still range 2 **without** indirect fire at gunpowder (so no change here, basically) 
3) range 3 **without** indirect fire at dynamite.

So, after experimenting with it a bit myself, I came to the following conclusion:
a) BombardRange needs to be updated each time BombardIndirect is (e.g., gunpowder); 
b) BombardIndirect needs to be updated each time BombardRange is (e.g., dynamite);
In other words, in order for it to work like I think it should, the code has to look like this:
UPDATE Technologies SET BombardRange=1 WHERE Type='TECH_AGRICULTURE';
UPDATE Technologies SET BombardRange=2 WHERE Type='TECH_MATHEMATICS';
**UPDATE Technologies SET BombardRange=2 WHERE Type='TECH_GUNPOWDER';**
UPDATE Technologies SET BombardIndirect=1 WHERE Type='TECH_GUNPOWDER';
UPDATE Technologies SET BombardRange=3 WHERE Type='TECH_DYNAMITE';
**UPDATE Technologies SET BombardIndirect=1 WHERE Type='TECH_DYNAMITE';**
